app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/TakePictureFragment.kt

```diff
@@ -146,6 +149,15 @@ class TakePictureFragment : Fragment() {
             })
     }
 
+    private fun applyImageSizeReduction(bytes: ByteArray): Uri {
+        val file = File(parentActivity.filesDir, "temp")
+        val fileOutputStream = FileOutputStream(file)
+        val bitmap = BitmapFactory.decodeByteArray(bytes, 0, bytes.size)
+        bitmap.compress(Bitmap.CompressFormat.JPEG, 25, fileOutputStream)
```

Comment:
maybe make the compression quality a constant in a companion object of the class, or a resource?

Reply:
true, thanks